data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/common/config/StreamingOptions.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.lambda.common.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class StreamingOptions {
+    @JsonProperty("enabled")
+    private boolean enabled = false;
+    
+    @JsonProperty("response_handling")
+    private String responseHandling = "reconstruct_document";
```

Comment:
Please make this an `enum` value. We have a standard convention for enums. Here is a decent example.

https://github.com/opensearch-project/data-prepper/blob/af304e65a28e6c8496d58c002f89c458aa85f1b1/data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/WhitespaceOption.java#L15

Use hyphens for the values. So this would have a value of `reconstruct-document`

```
public enum ResponseHandling {
  RECONSTRUCT_DOCUMENT("reconstruct-document");
...
```